app/helpers/application_cocoon_helper.rb

```diff
@@ -0,0 +1,53 @@
+module ApplicationCocoonHelper
+  # Shows a link that will allow to dynamically add a new associated object. This is a wrapper of
+  #   {#link_to_add_association}
+  #
+  # @overload link_to_association(name, form, options, association, html_options = nil)
+  #   Shows a link that will allow to dynamically add a new associated object.
+  #   @param [String] name the text to show in the link.
+  #   @param [FormBuilder] form the form builder.
+  #   @param [Symbol] association the associated objects, this should be the name of the has_many
+  #   relation.
+  #   @param [Hash] html_options the html options to be passed to {#link_to}.
+  #   @option html_options [Symbol] :selector the jquery selector of the node. Example
```

Comment:
'jQuery'

The jQuery selector to be used with +find_by+ to determine where to insert the new node.